chapters/rum/_toctree.yml

```
@@ -26,3 +26,22 @@
   - local: chapter1/10
     title: Quiz de final de capitol
     quiz: 1
+
+- title: 10. Curați seturi de date de înaltă calitate
```

```suggestion
- title: 10. Curățați seturi de date de înaltă calitate
```
Maybe `curați` is the right one, but seems off